tests/0080-admin_ut.c

```diff
@@ -694,6 +694,610 @@ static void do_test_DeleteConsumerGroupOffsets(const char *what,
         SUB_TEST_PASS();
 }
 
+/**
+ * @brief AclBinding tests
+ *
+ *
+ *
+ */
+static void do_test_AclBinding() {
+        int i;
+        char errstr[512];
+        rd_kafka_AclBinding_t *new_acl;
+
+        rd_bool_t valid_resource_types[]         = {rd_false, rd_false, rd_true,
+                                            rd_true,  rd_true,  rd_false};
+        rd_bool_t valid_resource_pattern_types[] = {
+            rd_false, rd_false, rd_false, rd_true, rd_true, rd_false};
+        rd_bool_t valid_acl_operation[] = {
+            rd_false, rd_false, rd_true, rd_true, rd_true, rd_true, rd_true,
+            rd_true,  rd_true,  rd_true, rd_true, rd_true, rd_true, rd_false};
+        rd_bool_t valid_acl_permission_type[] = {rd_false, rd_false, rd_true,
+                                                 rd_true, rd_false};
+        const char *topic     = test_mk_topic_name(__FUNCTION__, 1);
+        const char *principal = "User:test";
+        const char *host      = "*";
+
+        SUB_TEST_QUICK("AclBinding");
```

Comment:
nit: no need for the "AclBinding" string, just leave it empty `SUB_TEST_QUICK()`  since the function name is emitted and already contains the name.